x-pack/plugins/uptime/public/apps/uptime_overview_fetcher.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { fetchPingHistogram, fetchSnapshotCount } from '../state/api';
+import { UptimeFetchDataResponse } from '../../../observability/public/typings/fetch_data_response';
+
+export async function fetchUptimeOverviewData({
+  startTime,
+  endTime,
+  bucketSize,
+}: {
+  startTime: string;
+  endTime: string;
+  bucketSize: string;
+}) {
+  const snapshot = await fetchSnapshotCount({
```

Comment:
Don't you have to use the `bucketSize` here too?

Reply:
no, this is not a series

Reply:
it just returns you total number of monitors, number of up monitors and down monitors at that time.

Reply:
oh yeah, you're right!! thanks